Dockerfile

```diff
@@ -4,7 +4,12 @@ LABEL Maintainer="software-embedded-platform@ultimaker.com" \
       Comment="Ultimaker update-tools filesystem"
 
 RUN apk add --no-cache \
+        e2fsprogs \
+        f2fs-tools \
+        sed \
```

Comment:
i still think this is unnecessary for now as afaik the busybox variant does exactly what you request it to do.